src/main/python/rerank/scripts/utils.py

```diff
@@ -0,0 +1,122 @@
+import shlex
+import subprocess
+import sys
+import string
+printable = set(string.printable)
+printable.remove("\n")
+printable.remove("\t")
+printable.remove("\r")
+import os
+
+import jnius_config
+jnius_config.set_classpath("target/anserini-0.4.1-SNAPSHOT-fatjar.jar")
+from jnius import autoclass
+JString = autoclass('java.lang.String')
+JSearcher = autoclass('io.anserini.search.SimpleSearcher')
+
+def cal_score(fn_qrels="src/main/resources/topics-and-qrels/qrels.microblog2014.txt", prediction="score.txt"):
+    cmd = "/bin/sh run_eval_new.sh {} {}".format(prediction, fn_qrels)
```

Comment:
what is `run_eval_new.sh`?

Reply:
I'll check in it, or transform into python code.